man/chgpasswd.8.xml

```diff
@@ -88,12 +90,16 @@
 	<term><option>-c</option>, <option>--crypt-method</option></term>
 	<listitem>
 	  <para>Use the specified method to encrypt the passwords.</para>
-	  <para condition="no_sha_crypt">
-	    The available methods are DES, MD5, and NONE.
-	  </para>
-	  <para condition="sha_crypt">
-	    The available methods are DES, MD5, NONE, and SHA256 or SHA512
-	    if your libc support these methods.
+	  <para>
+	    The available methods are <phrase condition="bcrypt">
+	    <replaceable>BCRYPT</replaceable>,</phrase>
+	    <replaceable>DES</replaceable>,
+	    <replaceable>MD5</replaceable><phrase condition="sha_crypt">,
+	    <replaceable>SHA256</replaceable>,
```

Comment:
Similar here. Can you move the condition beginning to the same line as SHA256?

Reply:
If I do that, there will be a blank between `MD5` and the following comma, i.e. `MD5 ,` instead of `MD5,`. It is the same logic used as in ENCRYPT_METHOD.xml (see https://github.com/shadow-maint/shadow/blob/master/man/login.defs.d/ENCRYPT_METHOD.xml#L15).

So this comment applies to the other remarks regarding newlines as well.

Reply:
Ok, now I see why you decided to do it this way. Then, disregard this and other comments like this one.